### ISSUE TITLE
Tab switching

### ISSUE THREAD

**Issue by bbhenry** [oulvhai/MWeb-issues#272]
Can you make tab switching shortcut key function like the Atom IDE by pressing Command+Option＋(left or right) .

**Comment by oulvhai** [comment]
Please try Command + Shift + [  or  ] .

**Comment by bbhenry** [comment]
yes, it does work with the keys you show me, but it would be much nicer if it's the same shortcut key as the other apps that people use a lot (Atom code editor, Chrome browser). It will appeal much better with developers I think.

**Comment by oulvhai** [comment]
I think the OS X app more use Command + Shift + [ or ], for example: Safari, Xcode.
By the way, Chrome also use  Command + Shift + [ or ].
